websites/F/feds/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "1105810827224809532",
+		"name": "for"
+	},
+	"service": "feds",
+	"description": {
+		"en": "feds.lol is a freemium biolink service."
+	},
+	"url": "feds.lol",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/TzIxLtq.png",
+	"thumbnail": "https://i.imgur.com/Gh57ayn.jpg",
+	"color": "#000000",
```

Comment:
Shouldn't you use the blue that is also in the logo?

Reply:
The black matches the aesthetic more.